pkg/apis/config/features.go

```diff
@@ -70,6 +70,7 @@ func defaultFeaturesConfig() *Features {
 		PodSpecInitContainers:            Disabled,
 		PodSpecDNSPolicy:                 Disabled,
 		PodSpecDNSConfig:                 Disabled,
+		SecurePodDefaults:                Disabled,
```

Comment:
isn't this something we can do upstream?

Reply:
This is done upstream. There is a link in the description.